developer_manual/core/theming.rst

```diff
@@ -154,6 +154,53 @@ because the first is needed to enable translations in the JavaScript code and
 the second one is read by the PHP code and provides the data for translated
 terms in there.
 
+How to change names, slogans and URLs
+=====================================
+
+The ownCloud theming allows to change a lot of the names that are shown all over in the web interface. It's also possible to change the URLs to the documentation or the Android/iOS apps.
```

Comment:
Suggest:
The ownCloud theming allows a lot of the names that are shown on the web interface to be changed. It's also possible to change the URLs to the documentation or the Android/iOS apps.